### DISCUSSION TITLE
How to login with Windows domain accounts

### DISCUSSION THREAD

**Discussion by Mogaba** [ltsp/ltsp#287] [Q&A]
If LTSP server is a member of an Active Directory domain (joined using sssd), how do you login with domain accounts on the clients?

**Comment by maccadia** [comment]
This question has been raised in https://github.com/ltsp/ltsp/issues/16. There is currently no clean documented way to do that.

See :
 *  https://github.com/ltsp/community/wiki/LTSP-and-Samba-Windows-NT-Domain
 * https://github.com/ltsp/ltsp/issues/16#issuecomment-588127315
